orderly2outpack.R

```diff
@@ -3,15 +3,25 @@
 'orderly2outpack.
 
 Usage:
-  orderly2outpack.R <src> <dest> [--once]
+  orderly2outpack.R <src> <dest> [--once] [--custom=<schedule>]
 
 Options:
-  --once      Perform migration once.
+  --once                     Perform migration once.
```

Comment:
I think that docopt can be sensitive to whitespace issues - best to align these two options